### PR TITLE
lowlevel: add script to generate fake foot pressure values

### PR DESCRIPTION
**Pull request by jgueldenstein**
Adds a gui to generate fake data for foot pressure sensors.
The same zero and scale values may be reproduced by using the same random seed.

## Proposed changes
This tool will be useful in evaluating the pressure_converter.py and foot_pressure_calibration.py and the c++ versions without a robot.

## Necessary checks
- [x] Update package version
- [x] Run linters
- [x] Run `catkin build`
- [x] Write documentation
- [x] Test on your machine
- [ ] ~~Test on the robot~~

### CODE REVIEW

**Comment by jgueldenstein** [comment]
Some help would be appreciated to solve lines 38-71 and 81 more elegantly

File: bitbots_ros_control/scripts/foot_pressure_tester.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    force_values[5] = float(force)
+
+
+def update_force_6(force):
+    force_values[6] = float(force)
+
+
+def update_force_7(force):
+    force_values[7] = float(force)
+
```
Comment (jgueldenstein) [inline bitbots_ros_control/scripts/foot_pressure_tester.py]: I would like to solve this more elegantly but I could not find out how to do this in python.

Comment (SammyRamone) [reply bitbots_ros_control/scripts/foot_pressure_tester.py]: cant you just add a paramter for the number and then call it with the "i"

Comment (jgueldenstein) [reply bitbots_ros_control/scripts/foot_pressure_tester.py]: the problem is that I need to pass the function to the slider widget

**Comment by jgueldenstein** [comment]
merged for now, while the solution for the callback functions are not pretty, this can be solved later.

**Pull request merged**